arch/arm/boot/dts/overlays/revpi-core-overlay.dts

```diff
@@ -103,11 +103,14 @@
 				brcm,function = <BCM2835_FSEL_ALT2>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
-			sniff_a_pins: pb_sniff_a_pins {
-				/* A2 */
-				brcm,pins     = <28>;
+			sniff_pins: pb_sniff_pins {
+				/* 1A 2A 1B 2B */
+				brcm,pins     = <42 28 43 29>;
 				brcm,function = <BCM2835_FSEL_GPIO_IN>;
-				brcm,pull     = <BCM2835_PUD_DOWN>;
+				brcm,pull     = <BCM2835_PUD_OFF
+						 BCM2835_PUD_DOWN
```

Comment:
Should not this also be PUD_OFF?

Reply:
No as the 2A is the only pin without a external pull-up or pull-down. You added this as fix with commit 3b0119f8896e29cb97e5c9f7ba6daca3f80c1a7e. This only adds the other pins as well and makes sure no internal pull-up or -down is used on them.

Reply:
Yes right, I recall it, thx.